berkeley-function-call-leaderboard/data/possible_answer/BFCL_v3_live_multiple.json

```diff
@@ -933,7 +933,7 @@
 {"id": "live_multiple_932-191-20", "ground_truth": [{"view_service_provider_profile": {"professional_id": [4724]}}]}
 {"id": "live_multiple_933-191-21", "ground_truth": [{"get_service_providers": {"avg_rating": ["", null], "province_id": [1], "district_name": ["Ratchathewi"], "sub_district_name": ["", null], "start_available_date": ["2024-03-20 11:00:00"], "end_available_date": ["", null], "min_age": ["", null], "max_age": ["", null], "has_quality_problem": ["", false], "has_late_check_in": ["", false], "is_excellent": ["", false], "is_package": ["", false], "is_subscription": ["", false], "service_id": ["", null], "available_for_pet": [true], "professional_group_id": ["", null], "job_qty": ["", null], "is_cleaning_condo": ["", false], "is_cleaning_home": ["", false], "is_cleaning_office": ["", false]}}]}
 {"id": "live_multiple_934-191-22", "ground_truth": [{"get_service_providers": {"avg_rating": ["", null], "province_id": [1], "district_name": ["Chatuchak"], "sub_district_name": ["", null], "start_available_date": ["2024-03-22 00:00:00"], "end_available_date": ["2024-03-22 23:59:59"], "min_age": ["", null], "max_age": ["", null], "has_quality_problem": ["", false], "has_late_check_in": ["", false], "is_excellent": [true], "is_package": ["", false], "is_subscription": ["", false], "service_id": [1], "available_for_pet": ["", false], "professional_group_id": ["", null], "job_qty": ["", null], "is_cleaning_condo": ["", false], "is_cleaning_home": ["", false], "is_cleaning_office": ["", false]}}]}
-{"id": "live_multiple_935-192-0", "ground_truth": [{"set_countdown": {"duration": ["01:00"], "purpose": ["meeting"]}}]}
```

Comment:
can we do something better here? for example checking that the string has "meeting" rather very specific case-specific strings?

Reply:
As mentioned in #643, unfortunately, we don't have any semantic checkers for the parameter values at this moment. We will explore that option and keep you updated.

Reply:
Ok, thanks. Suggestion: while evaluating model performance, can we lowercase all param values, given that is what the ground truth is restricted to it seems?

Reply:
Yes. This is the current strategy we use. When the parameter value is a string, we compare the lowercase version of the model response with the lowercase version of the ground truth. Code [here](https://github.com/ShishirPatil/gorilla/blob/main/berkeley-function-call-leaderboard/bfcl/eval_checker/ast_eval/ast_checker.py#L166-L191).